src/main/java/com/ibm/as400/access/Permission.java

```diff
@@ -722,6 +721,23 @@ public Enumeration<String> getAuthorizedUsers()
       }
     }
 
+    /**
+     * Returns a List of authorized users.
+     * @return A list of authorized users.
+     *
+     **/
+    public List<String> getAuthorizedUsersList() {
```

Comment:
Since we'll have your new-and-improved `List` flavor, we should rewrite the implementation of `getAuthorizedUsers()` to leverage `Collections.enumeration()`, for instance 
```
return Collections.enumeration(getAuthorizedUsersList());
```
This saves redundant implementation, and the implementation of `Collections.enumeration` is very efficient

Reply:
thanks, updated